runtime/integration-tests/src/xcm/development/tests/connectors.rs

```diff
@@ -270,6 +285,19 @@ fn encoded_ethereum_xcm_add_pool() {
 	assert_eq!(encoded_call_hex, expected_encoded_hex);
 }
 
+// Verify that the max tranche token symbol and name lengths are what the Connectors pallet expects.
+#[test]
+fn verify_tranche_fields_sizes() {
+	assert_eq!(
+		development_runtime::MaxTokenNameLength::get(),
+		pallet_connectors::TOKEN_NAME_SIZE as u32
+	);
+	assert_eq!(
+		development_runtime::MaxTokenSymbolLength::get(),
+		pallet_connectors::TOKEN_SYMBOL_SIZE as u32
+	);
+}
```

Comment:
@offerijns @mikiquantum my goal with this sanity test is to ensure that we don't change one of the size limits (either on `PoolSystem` or on `Connectors`) silently.

Another approach we can follow here is to have an `ensure!(` mark for the `name` and `symbol`s to ensure they are not bigger than what's expected and fail with the `InvalidTrancheMetadataName` and `InvalidTrancheMetadataSymbol`; however, if the two pallets agree at compile time that they share the same limits, that check is effectively added noise.

What do you think?